packages/pdfparser.vm/pdfparser.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>pdfparser.vm</id>
+    <version>0.7.7</version>
+    <description>pdf-parser.py: a solid script to extract objects from PDF files</description>
+    <authors>Didier Stevens</authors>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="python3" version="(, 3.10.4]" />
```

Comment:
Should this depend on `libraries.python3.vm` or a custom Python3 package? I can imagine errors around various Python versions (and keeping them in sync in packages).

Not sure if the CI error is related exactly to that:
```
WARNING: Generic MSI Error. This is a local environment error, not an issue with a package or the MSI itself - it could mean a pending reboot is necessary prior to install or something else (like the same version is already installed). Please see MSI log if available. If not, try again adding '--install-arguments="'/l*v c:\python3_msi_install.log'"'. Then search the MSI Log for "Return Value 3" and look above that for the error.
ERROR: Running ["C:\ProgramData\chocolatey\lib\python3\tools\python-3.10.4-amd64.exe" /quiet InstallAllUsers=1 PrependPath=1 TargetDir="C:\Python310" ] was not successful. Exit code was '1603'. Exit code indicates the following: Generic MSI Error. This is a local environment error, not an issue with a package or the MSI itself - it could mean a pending reboot is necessary prior to install or something else (like the same version is already installed). Please see MSI log if available. If not, try again adding '--install-arguments="'/l*v c:\python3_msi_install.log'"'. Then search the MSI Log for "Return Value 3" and look above that for the error..
The install of python3 was NOT successful.
```

Reply:
Good catch! I see two options:

1. When a package needs Python, depend on `libraries.pythonX.vm`. Downside is it also installs all the Python libraries which may or may not be wanted.
2. Create a new metapackage to install a specific version of PythonX to unify / simply the global VM Python version. Pro is that we can have `libraries.pythonX.vm` depend on this new metapackage as well as other packages that want to utilize Python as well. Con is if we need multiple versions of Python (e.g., 3.7 and 3.10) for whatever reason.

I'm in favor of option `2` above. Thoughts?

Reply:
2. sounds good! Ideally, we can focus on one core version and let users handle any special use-cases.

There's also pyenv, but I have no experience with it and it does not officially support Windows.

Reply:
Works for me, I'll get this up pretty soon with a PR, then we can circle back to this package

Reply:
Added with #263

Reply:
Welp, I added it but I need to fix it. I got too hasty. Will post back once fixed